### ISSUE TITLE
Voting seems to be broken; voting as if a vote by all players, not just yourself

### ISSUE THREAD

**Issue by zedwick** [stefvanschie/buildinggame#64]
Players voting will vote for current and previously voted on plots as if by all players in the game.

Voting 7 on plot 2, if you were plot 1 and gave yourself an 8; This is just one vote:

```
You can't vote on your own plot
You gave Dax23333's build a 7
You can't vote on your own plot
You gave Dax23333's build a 7
You can't vote on your own plot
You gave Dax23333's build a 7
You can't vote on your own plot
You gave Dax23333's build a 7
You can't vote on your own plot
You gave Dax23333's build a 7
```

This set's Dax's points to 7 in the scoreboard, as well as my own to 7.
If Dax, from plot 2, had voted on my plot first, and I had not, (giving it a 4, I am terrible) and I then voted to give his a 7 as above, then both my score and his would be 7 resulting in me winning the round at the end.

Also note that if either of us vote on plot 1, then only plot 1s score is affected. But if plot 1 already has a point score, then voting on plot 2's score will alter plot 1's.

BuildingGame version: 4.0.6

**Comment by stefvanschie** [comment]
Will fix it in the next version

**Comment by stefvanschie** [comment]
Fixed in version 4.0.7
